opensearch/src/test/java/org/opensearch/sql/opensearch/storage/script/filter/lucene/MatchPhraseQueryTest.java

```diff
@@ -30,20 +30,27 @@ public class MatchPhraseQueryTest {
 
   private final DSL dsl = new ExpressionConfig().dsl(new ExpressionConfig().functionRepository());
   private final MatchPhraseQuery matchPhraseQuery = new MatchPhraseQuery();
-  private final FunctionName matchPhrase = FunctionName.of("match_phrase");
+  private final FunctionName matchPhraseName = FunctionName.of("match_phrase");
+  //TODO: Consider also testing 'matchphrase'
```

Comment:
TODO?

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/165/commits/d04916bfbe558347bec18a9b61b3f13697ce270d